documentation/release_notes.markdown

```diff
@@ -4,271 +4,39 @@ title: "Puppet Server: Release Notes"
-
-#### Added setting to flush ruby instances after a configurable number of requests
-
-We added a setting that can be used by the master to limit how many HTTP requests a given JRuby instance will handle in its lifetime. When a JRuby instance reaches this limit, it is flushed from memory and replaced with a fresh one. Defaults to 0, which disables automatic JRuby flushing. This can be useful for working around buggy module code that would otherwise cause memory leaks, however _it causes a slight performance penalty_ whenever a new JRuby has to reload all of the Puppet Ruby code. If memory leaks from module code are not an issue in your deployment, the default value will give you the best performance.
-
-* [SERVER-325](https://tickets.puppetlabs.com/browse/SERVER-325)
-
-#### Allow `environment-cache` to take an environment as an argument
-
-Added support to the `environment-cache` API for flushing an environment by name, as opposed to only having the ability to flush all environments.
-
-* [SERVER-324](https://tickets.puppetlabs.com/browse/SERVER-324) 
+This is a security and bug fix release of Puppet Server 1.1 series.  No new
+features have been added since 1.1.0 in accordance with [Semantic
+Version](http://semver.org) guidelines.
 
 ### Bug fixes
 
-#### Re-enabled the master `status` endpoint
-
-* [SERVER-564](https://tickets.puppetlabs.com/browse/SERVER-564)
-
-#### `ignore` parameters were being mishandled
-
-Fix for a problem where `file_metadatas` requests to the master which include multiple `ignore` parameters were being mishandled. This had previously led to an agent downloading files from the master which should have been ignored.
-
-* [SERVER-442](https://tickets.puppetlabs.com/browse/SERVER-442)
-* [SERVER-696](https://tickets.puppetlabs.com/browse/SERVER-696)
-
-#### `keylength` value now being determined by setting
+#### Upgrade JRuby from 1.7.20 to 1.7.20.1 to resolve CVE-2015-4020
 
-Previously, Puppet Server had always hardcoded the `keylength` to 4096 bits. Now, it properly honors the value in the `keylength` setting in puppet.conf when determining the number of bits to use in the generation of keys.
-* [SERVER-157](https://tickets.puppetlabs.com/browse/SERVER-157)
+[SERVER-761](https://tickets.puppetlabs.com/browse/SERVER-761) Bump JRuby
+dependency to version 1.7.20.1
 
-#### Verbose output disabled
+#### Consolidate environment handling behavior
 
-Disabled the display of verbose output that appeared during a package upgrade.
+[SERVER-297](https://tickets.puppetlabs.com/browse/SERVER-297) Consolidate JRuby
+environment handling, which was previously inconsistent across the use cases of
+`puppetserver gem`, `puppetserver irb`, `puppetserver ruby` and the puppetserver
+service.
 
-* [SERVER-541](https://tickets.puppetlabs.com/browse/SERVER-541)
+#### Return good Content-Type for CA errors
 
-#### Previous logback level issue fix had been reverted
+Two bugs in the way the CA service response which caused issues for services
```

Comment:
Great catch, thanks, fixed in 3e07335 `(SERVER-749) Update release notes for 1.1.1`